help/destinations/catalog/advertising/tradedesk-emails-update

```diff
@@ -0,0 +1,138 @@
+---
+title: (Beta) The Trade Desk - CRM connection
+description: Activate profiles to your Trade Desk account for audience targeting and suppression based on CRM data.
+exl-id: e09eaede-5525-4a51-a0e6-00ed5fdc662b
+---
+# (Beta) The [!DNL Trade Desk] - CRM connection
+
+>[!IMPORTANT]
+>
+>[!DNL The Trade Desk - CRM] destination in Platform is currently in beta. The documentation and functionality are subject to change. 
+>
+>With the release of EUID (European Unified ID), you are now seeing two [!DNL The Trade Desk - CRM] destinations in the Destinations catalog.
+>* if you source data in the EU, please use [!DNL The Trade Desk - CRM (EU)] destination. 
+>* if you source data in the APAC or NAMER regions, please use [!DNL The Trade Desk - CRM (NAMER & APAC)] destination. 
```

Comment:
Two comments here: 

Are you planning to rename the "old" Tradedesk CRM destination to The Trade Desk - CRM (NAMER & APAC)? In that case, you should open a separate publishing request to update the name of that destination.

On a second note (and this is just a suggestion), I'm not sure that the NAMER acronym is easily understood. In a brief Google search, I found NAM as a more common acronym for the North Americas region. Perhaps you'd want to update the acronym.

Reply:
@vgiurgiu in our email thread, EJ confirmed we will be moving forward with the NAMER acronym and we will be submitting a separate publisher request update the name of the original destination. This will not impact documentation updates.